index.js

```diff
@@ -55,6 +57,15 @@ export const run = async () => {
         output = readFileSync(content).toString("utf-8");
     }
 
+    if (typeof replacementRegex === "object" || replacementRegex.length > 0) {
```

Comment:
This checks if the replacement is not an empty string (at least 1 character or an RegExp object). This check can save resources by not executing the code if no replacement Regex is set.